nixos/default.nix

```diff
@@ -44,10 +38,12 @@ let
       powerManagement = stub;
       security.pam.usb = stub;
       security.pam.mount = stub;
+      security.pam.oath = stub;
       boot.isContainer = lib.mkOption { default = false; };
       boot.initrd.luks = stub;
       networking.wireless = stub;
       networking.connman = stub;
+      virtualisation.vswitch = stub;
```

Comment:
Why do we need these stubs? If I'm reading this correctly they disable the services. Does that imply these services are enabled by default and we want to turn them off?